packages/app/src/config/pages.ts

```diff
@@ -52,10 +52,10 @@ export const PagesConfig: PagesConfigItems = [
   },
   {
     category: 2,
-    key: 'payouts',
-    uri: `${BASE_URL}payouts`,
-    hash: '/payouts',
-    Entry: Payouts,
+    key: 'rewards_payouts',
```

Comment:
To keep titles concise, let's just use `rewards` 
```suggestion
    key: 'rewards',
```